pkg-py/docs/index.qmd

```diff
@@ -45,6 +44,7 @@ from querychat.data import titanic
 
 qc = QueryChat(titanic(), "titanic", client="openai/gpt-4.1")
 app = qc.app()
+# app.run()
 ```
 
 With the above code saved to `titanic-app.py` and an API key set[^api-key], you can [run the app](https://shiny.posit.co/py/get-started/create-run.html#run-your-shiny-application) from a terminal (or [VSCode](https://marketplace.visualstudio.com/items?itemName=Posit.shiny)):
```

Comment:
Is it VSCode or VS Code? Turns out there's [a right answer](https://code.visualstudio.com/brand): `VS Code`.

Also, I think we should mention Positron first everywhere it comes up and add `(or VS Code)` if we need to mention VS Code.